roles/libvirt_manager/tasks/clean_layout.yml

```diff
@@ -2,10 +2,19 @@
 - name: Get installed packages list
   ansible.builtin.package_facts: {}
 
-- name: Get virtqemud socket
-  register: _virtqemud
-  ansible.builtin.stat:
-    path: "/var/run/libvirt/virtqemud-sock"
+- name: Populate service facts
+  ansible.builtin.service_facts:
+
+- name: Start virtqemud socket service
+  ansible.builtin.service:
+    name: "{{ item }}"
+    state: started
+    enabled: yes
+  loop:
+    - virtqemud.service
+    - virtqemud.socket
+  when: ansible_facts['services']['virtqemud.service']['status'] | default('not-found') != 'not-found'
```

Comment:
This line is not required. Based on [the module description](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/service_module.html#parameter-state) the start action will be executed only when it's not already running.
> `started`/`stopped` are idempotent actions that will not run commands unless necessary.

Reply:
true but it is not true if it is not installed/doesn't exist

Reply:
I checked it out locally and am surprised myself